app/static/js/events/scheduler.js

```diff
@@ -849,6 +848,10 @@ function loadDateButtons() {
  */
 function loadMicrolocationsToTimeline(day) {
 
+    $('table.timeline-table').show();
```

Comment:
Move all selectors to the above (https://github.com/SaptakS/open-event-orga-server/blob/9c1f7b9f57ee26220ecd27825f818c7269e0e8bc/app/static/js/events/scheduler.js#L123) for consistency